src/nvim/mark.c

```diff
@@ -1499,12 +1501,13 @@ void copy_viminfo_marks(vir_T *virp, FILE *fp_out, int count, int eof, int flags
       if (load_marks) {
         if (line[1] != NUL) {
           int64_t lnum_64;
-          unsigned u;
+          unsigned int u;
```

Comment:
Well, you said it all ;-)
In new/refactored code, we try to use the complete type name.